Interface.md

```diff
@@ -42,5 +42,13 @@ services through the exploitation of standard protocols and the harmonizing of s
 | **AUTH\#1** |  Authentication interfaces should support the [OpenID Connect](https://openid.net/developers/how-connect-works/) Protocol. |
 | **AUTH\#2** |  HTTP requests (e.g. for data access) subject to authorization shall include the user token with `claims` in [JWT](https://datatracker.ietf.org/doc/html/rfc7519) format returned by the Authentication interface with every API request as a bearer token in the HTTP authorization header. |
 
+
+Discussion Points/questions for the group:
+
+- Do we want to recommend that Agencies document the API (interfaces) consumed by their users (Discovery, Data Access etc.) via executable Jupyter Notebooks ?
```

Comment:
They shouldn't need to, in my view. If agencies are providing a STAC API and publicly accessible files as assets, then users can use generic tools to access them.

That said, providing notebooks is a great way to help users understand the data and things like masking clouds or other data-specific processing users can do.

Reply:
> I think the recommendation should be that the API is documented (mandatory)

I definitely agree, but would probably go further and say "unless absolutely necessary, the API should conform to existing open specifications/standards like OGC API specifications and OGC STAC API Community Specification.

> ideally, that the documentation includes examples of the requests/payloads and flows

Agree too, and if the API is an open specification, the examples should already exist. We can always do with more documentation and examples though!

Reply:
Note: STAC is not yet an OGC community specification.

Reply:
Status: https://www.ogc.org/requests/ogc-looks-to-adopt-stac-as-community-standard-seeking-public-comment-before-moving-to-vote/